src/ypd/templates/project.html

```diff
@@ -28,6 +28,16 @@ <h1>{{project.title}}</h1>
         </div>
     {% endif %}
 
+    {% if project.can_be_modified_by(user) %}
+        <div class="projectButton">
+            <a href="{{url_for('ProjectView:archive', id=project.id, is_provided=(project is provided))}}">
+                <button>
+                    🗑️
+                </button>
+            </a>
+        </div>
+    {% endif %}
+
```

Comment:
How can you tell if a project has been archived just by looking at it?

Reply:
Added two icons, one to archive flag true and one to archive flag false.